README.md

```diff
@@ -69,6 +78,13 @@ Everything you need will be installed locally. Simply run:
 make init
 ```
 
+Perform a one-off configuration of your local Amazon CLI client by executing:
+```bash
+ansible/.venv/bin/aws --profile nhsd-profile-name configure
+```  
+This will ask you to provide your AWS access key id and access key value, AWS region
+(`eu-west-1`) and preferred [format of output generated by the client](https://docs.aws.amazon.com/cli/latest/userguide/controlling-output.html).
+It then saves those details in `~/.aws/configuration` and `~/.aws/credentials`. The name of the profile (``)
```

Comment:
👍

Reply:
Is this sentence finished? 😃

Reply:
Good spot!

Unusually, my IDE was giving me some problems when editing Markdown, for some reason,  and it seems I've lost some of the content.

Now fixed.